packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dashboard.html

```diff
@@ -93,19 +48,62 @@
                             </p>
                         </div>
                     </div>
-                    <div class="oui-tile__actions">
-                        <oui-action-menu data-compact data-placement="end">
-                            <oui-action-menu-item
-                                data-href="#/useraccount/infos"
-                            >
-                                <span
-                                    data-translate="user_account_dashboard_profile_edit"
-                                ></span>
-                            </oui-action-menu-item>
-                        </oui-action-menu>
+                    <div class="oui-tile__definition">
+                        <a
+                            class="oui-button oui-button_secondary mx-auto my-5 d-block"
+                            data-ng-href="#/useraccount/infos"
+                            rel="noopener"
```

Comment:
**question:** Still necessary?

Reply:
indeed, the div was necessary because otherwise the button could not be refocused and take the width

Reply:
indeed after analysis, the rel=noopener is not necessary. i took it off